content/en/guide/v10/typescript.md

```diff
@@ -7,8 +7,6 @@ description: "Preact has built-in TypeScript support. Learn how to make use of i
 
 Preact ships TypeScript type definitions, which are used by the library itself! 
 
-When you use Preact in a TypeScript-aware editor (like VSCode), you can benefit from the added type information while writing regular JavaScript. If you want to add type information to your own applications, you can use [JSDoc annotations](https://fettblog.eu/typescript-jsdoc-superpowers/), or write TypeScript and transpile to regular JavaScript. This section will focus on the latter.
```

Comment:
This is a description of the VSCode features and methods of adding types.
I think it is unnecessary in Preact document.
Therefore, I removed this line.

Reply:
A thought: may we could have this just be a short "tip" like this:

<img width="869" alt="Screen Shot 2020-12-07 at 10 31 46 AM" src="https://user-images.githubusercontent.com/105127/101370389-70395000-3877-11eb-9b93-3d79a2729f22.png">

the markup I used there was:

```md
> 💁 You can also use Preact's type definitions in plain JavaScript with [JSDoc annotations](https://fettblog.eu/typescript-jsdoc-superpowers/).
```